src/server/trpc/router/loginId.ts

```diff
@@ -1,17 +1,18 @@
+import { TRPCError } from "@trpc/server";
+import { type Session } from "next-auth";
 import { addLogin, readLoginList } from "utils/database/loginID.db";
 import { z } from "zod";
 
 import { router, publicProcedure } from "../trpc";
 
 export const loginRouter = router({
-  list: publicProcedure.query(async () => {
```

Comment:
Since authntication is check on the server, we do not need to expose the user list

Reply:
That's great. Also only allowed to use the add function if authenticated.